solution-base/build.sh

```diff
@@ -138,6 +150,59 @@ function kafka_yamls()
     echo --- >> ${KAFKA_OPERATOR_PATH}
 }
 
+function mongodb_yamls()
+{
+    OUTPUT_PATH=${1:-${OPERATOR_PATH}}
+    NODE_COUNT=${2:-1}
+
+    echo merging mongodb yamls
+    CHART_PATH="$(dirname $0)/mongodb/charts/mongodb"
+    
+    echo "${CHART_PATH}/values-production.yaml" 
+    if [[ "$NODE_COUNT" == "3" ]]; then
+        helm template ${MONGODB_NAME} ${CHART_PATH} -n ${MONGODB_NAMESPACE} \
+            -f "${CHART_PATH}/values-production.yaml" \
+            --set "separateSecrets=true" \
+            --set 'existingUserSecret=$USER_SECRET_PLACEHOLDER' \
+            --set 'existingRootSecret=$ROOT_SECRET_PLACEHOLDER' \
```

Comment:
could this placeholder possibly be substituted in by the artesca-installer

Reply:
It seems in the upgrade case, the secret would already exist (although might need to be copied in case it's garbage collected), and in the fresh install case, the secret would need to be create by the installer.